stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -2657,8 +2657,10 @@ function _spsetnz_setindex!(A::AbstractSparseMatrixCSC{Tv}, x::Tv,
                     resize!(nzvalA, nnzA)
                 end
                 r = rowidx:(rowidx+nincl-1)
-                rowvalA[r] = I
-                nzvalA[r] = x
+                rowvalA[r] .= I
+                for rr in r
+                    nzvalA[rr] = x
+                end
```

Comment:
It seems like
```suggestion
                nzvalA[rr] .= Ref(x)
```
would work also in the case when `x` is a matrix. Obviously just a matter of style, not of correctness...

Reply:
Do you want me to change this or can this be merged?

Reply:
As you like. I think we can merge this as is.